frontend/src/entities/jobs/hpo/create/training-definitions/algorithm-options.tsx

```diff
@@ -436,7 +436,7 @@ export function AlgorithmOptions (props: AlgorithmOptionsProps) {
                                 },
                             },
                         ]}
-                        empty='No algorithm selected'
+                        empty='There are currently no resources.'
```

Comment:
I'm assuming this is the string used in the console? Should this just say `No metrics for available for selected algorithm` if an algorithm is selected and keep the old message if no algorithm is selected? I'm not totally sure I'd know what `no resources` means in this context.